packages/node_modules/@webex/webex-core/test/unit/spec/services/service-catalogs.js

```diff
@@ -152,6 +153,48 @@ describe('webex-core', () => {
       });
     });
 
+    describe('#failed()', () => {
+      let filter;
+      let host;
+
+      beforeEach('generate the service host class objects', () => {
+        serviceCatalogs.load(ServiceCatalogs.mapRemoteCatalog({
+          catalog: SERVICE_CATALOGS[0],
+          ...fixture
+        }));
+
+        host = serviceCatalogs.hosts[0];
```

Comment:
Could we call this `filteredHost` since it is the expected host to have been filtered in the test? Makes the `it` blocks a bit easier to comprehend.

Reply:
I will change this to `hostToFilter` so that this test doesn't stray far from how the other tests handle a `filteredHost` namespace.